packages/common-all/src/types/typesv2.ts

```diff
@@ -538,3 +549,7 @@ export enum DendronTreeViewKey {
   TREE_VIEW_V2 = "dendron.tree-view",
   BACKLINKS = "dendron.backlinks",
 }
+
+export enum DendronCalendarViewKey {
```

Comment:
instead of a separate enum, lets use the existing `DendronTreeViewKey`

Reply:
f435c5d3